<ui version="4.0" >
 <class>RemovePhrasesByNameDialogClass</class>
 <widget class="QDialog" name="RemovePhrasesByNameDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>141</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>490</width>
    <height>141</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>490</width>
    <height>141</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Удалить все фразы текущей разметки с заданным именем</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="1" >
    <widget class="QLineEdit" name="lineEdit_phrasesName" />
   </item>
   <item row="4" column="1" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Имя фразы:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QRadioButton" name="radioButton_removeOnlyPhrases" >
     <property name="text" >
      <string>Удалить только фазы (без смещения сосодних границ).</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QRadioButton" name="radioButton_removePhrasesAndSignal" >
     <property name="text" >
      <string>Удалить фразы вместе с сигналом фонограммы.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QCheckBox" name="checkBox_caseSensitive" >
     <property name="text" >
      <string>Поиск фраз с учётом регистра</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RemovePhrasesByNameDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>322</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RemovePhrasesByNameDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>390</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
